Robot Code/robot.py

```diff
@@ -52,23 +52,20 @@ def robotInit(self):
 
     def autonomousPeriodic(self):
         self.automodes.run()
-        pass
 
 
     def teleopPeriodic(self):
 
 
         self.drive.tankDrive(self.xboxController.getY(), self.xboxController.getRawAxis(5))
        
-       
-        if self.xboxYbutton.get():
-            self.drive.flipflip()
-     
+]
         if self.xboxAbutton.get():
```

Comment:
I thought we decided to get rid of mode switching in teleop?

Reply:
We did, but popular vote wanted it back